pcdsutils/requirements.py

```diff
@@ -213,3 +218,4 @@ def _compare_requirements(args=None):
                     else:
                         logger.info('- %s', item)
                 logger.info('')
+    return 1 if has_diff else 0
```

Comment:
```suggestion
    return 1 if has_diff else 0
    
```

Reply:
Good catch.. I always forget that... 😞

Reply:
Very easy to forget. I would forget it all the time if my editor didn't automatically check.

Reply:
Oops, I guess there was some whitespace in there.

Reply:
I just pushed a fix...